collection/rancher/v2.x/usage-collector/rancher2_usage_collector.sh

```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
 
 echo "Report run on `date`"
-echo ""
+echo
 
 # Get docker id for rancher single node install
 DOCKER_ID=$(docker ps | grep "rancher/rancher:" | cut -d' ' -f1)
```

Comment:
Thoughts on using something like `DOCKER_ID=$(docker ps -q --filter ancestor=rancher/rancher)` to cover single/HA in one command?

Issue: when using `rancher/rancher:stable` with helm for some reason the `docker ps` output uses the image ID, not the name which will break this.

```
CONTAINER ID        IMAGE                                   COMMAND                  CREATED             STATUS              PORTS               NAMES
50e9ec52d0e0        c583c217e121                            "entrypoint.sh --no-…"   6 seconds ago       Up 5 seconds                            k8s_rancher_rancher-656c99566c-ckndl_cattle-system_78cc5a67-8179-43bd-9b59-e87625b3b84e_0
```

Reply:
@dkeightley thanks for the review and the commit. Yes, would be great to be able to use a single command, but sounds like it breaks when using HA with stable tag?

Reply:
Yes, looks like keeping the existing approach will be more reliable.